feature-libs/product-configurator/common/components/configurator-cart-entry-bundle-info/configurator-cart-entry-bundle-info.component.html

```diff
@@ -0,0 +1,49 @@
+<ng-container *ngIf="orderEntry$ | async as orderEntry">
+  <ng-container *ngIf="isBundleBasedConfigurator(orderEntry)">
+    <ng-container *ngIf="(numberOfLineItems$ | async) > 0">
```

Comment:
`*ngIf="numberOfLineItems$ | async as numOfItems` (no need zero check, since if the value is 0, then *ngIf returns false). The benefit of doing this is: no need to subscribe `numberOfLineItems$` twice. In this line `| cxTranslate: { count: numberOfLineItems$ | async }`, we can use this `numOfItems` directly.

Reply:
Pick!

Reply:
Done!